showcases/e2e/switch/showcase-switch.spec.ts

```diff
@@ -3,5 +3,9 @@ import { test } from '@playwright/test';
 import { getDefaultScreenshotTest } from '../default.ts';
 
 test.describe('DBSwitch', () => {
-	getDefaultScreenshotTest({ path: '03/switch' });
+	getDefaultScreenshotTest({
+		path: '03/switch',
+		// It's an issue in the tool: https://github.com/IBMa/equal-access/issues/842
+		aCheckerDisableRules: ['aria_attribute_valid']
+	});
```

Comment:
```suggestion
	getDefaultScreenshotTest({ path: '03/switch' });
```

https://github.com/IBMa/equal-access/issues/842 seems to have been fixed

Reply:
I think it isn't published...

Reply:
opened https://github.com/db-ui/mono/issues/2861